### PR TITLE
Added mock task

### PR DESCRIPTION
**Pull request by awolkers**
The mock task watches the mock folder and copies files add/modified within this folder.

### CODE REVIEW

**Comment by Thomas-Machielsen** [comment]
Hi everybody, I feel like the current example (the waffle image - which looks tasty) doesn't really show what you'd usually mock, e.g. API data. If we can add a little function that fetches mock data I think we more clearly show what the mock folder is used for.

**Comment by awolkers** [comment]
I added an `example.json` mock file which is used by the Nav component.

File: src/components/nav/nav.njk [hunk 1]
```diff
@@ -1,3 +1,3 @@
-<nav data-module="nav/Nav">
+<nav data-module="nav/Nav" data-url="../mock/example.json">
```
Comment (waarissyb) [inline src/components/nav/nav.njk]: This should be a root-relative URL in order to make this example work in the component documentation (demo)

File: src/components/nav/_nav.scss [hunk 2]
```diff
@@ -1,4 +1,4 @@
 nav {
-  transform: translate(50%);
```
Comment (waarissyb) [inline src/components/nav/_nav.scss]: 👍

**Pull request merged**